beacon-chain/forkchoice/doubly-linked-tree/node.go

```diff
@@ -127,6 +127,10 @@ func (n *Node) setNodeAndParentValidated(ctx context.Context) error {
 	return n.parent.setNodeAndParentValidated(ctx)
 }
 
+func (n *Node) secondsSinceSlotStart(genesisTime uint64) uint64 {
+	return n.timestamp - genesisTime - uint64(n.slot)*params.BeaconConfig().SecondsPerSlot
```

Comment:
This feels like it definitely belongs in the `slots` package rather than here

Reply:
There is such a helper function in the `slots` package, that takes a time. instead of calling this as a method in the node I could have the caller using that helper function and passing it the genesis time and the node's timestamp. I wanted to avoid the parameter passing, but if you think it's better to use that method I can do that.